packages/TorneloScoresheet/src/components/Toolbar/ToolbarButton.tsx

```diff
@@ -0,0 +1,35 @@
+import { SvgProps } from 'react-native-svg';
+import { ColourType } from '../../style/colour';
+import RoundedView from '../RoundedView/RoundedView';
+import React from 'react';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+
+export type ToolbarButtonProps = {
+  Icon: React.FC<SvgProps>;
+  onPress: () => void;
+  colour: ColourType;
+  display: 'none' | undefined;
+};
+
+const ToolbarButton: React.FC<ToolbarButtonProps> = ({
+  Icon,
+  colour,
+  display,
+  onPress,
+}) => {
+  return (
+    <TouchableOpacity onPress={() => onPress()} style={{}}>
+      <RoundedView
```

Comment:
We actually already have a `ActionButton` component that we use for the action buttons, 
It accepts a svg file for the icon, 
text for the name,
a function to trigger on press

I think we should resuse that component instead of creating a new one since its basically the same thing,
The only edit needed is to change the styling, (right now the button has a blue background and the icon and text are white) you can add a boolen parameter called 'invert_colours' or something that would render a white button with blue interiors

Reply:
Edit: You'll also need a visible property to show/hide it, it can be optional and shown by default to not affect the current invocations